strawberryfields/engine.py

```diff
@@ -603,21 +604,64 @@ def run_async(self, program: Program, *, compile_options=None, **kwargs) -> Job:
         Keyword Args:
             shots (Optional[int]): The number of shots for which to run the job. If this
                 argument is not provided, the shots are derived from the given ``program``.
+            recompile (bool): specifies if ``program`` should be recompiled
```

Comment:
Just want to double check with either @heltluke or @lneuhaus: does adding SF specific `eng.run()` keyword arguments here have any implications for HW-specific keyword arguments on your end?

Reply:
It is something we'll need to prepare for, as we presently do some validation of our own, but I don't think it's much of an issue.

Could we coordinate the release of SF that will include this change with an appropriate update to gbsdriver?

Reply:
:+1: Perhaps what we should do on our end is 'remove' all SF-specific keywords from `kwargs` before blackbird serialization

Reply:
That could also work. :)